src/views/download/download.jsx

```diff
@@ -0,0 +1,180 @@
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                        <li className="installation-downloads-item">
+                                            <FormattedMessage id='download.linux' /> -
+                                            {' '}<a href="http://airdownload.adobe.com/air/lin/download/2.6/AdobeAIRInstaller.bin">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+                                <div className="installation-column">
+                                    <h3><FormattedMessage id='download.supportMaterialsTitle' /></h3>
+                                    <p><FormattedMessage id='download.supportMaterialsBody' /></p>
+                                    <ul className="installation-downloads">
+                                        <li className="installation-downloads-item">
+                                            <FormattedMessage id='download.starterProjects' /> -
+                                            {' '}<a href="https://scratch.mit.edu/scratchr2/static/sa/Scratch2StarterProjects.zip">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                        <li className="installation-downloads-item">
+                                            <FormattedMessage id='download.gettingStarted' /> -
+                                            {' '}<a href="https://cdn.scratch.mit.edu/scratchr2/static/__709da8e5f3d72129538a4ccdbcbf5f2a__/pdfs/help/Getting-Started-Guide-Scratch2.pdf">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                        <li className="installation-downloads-item">
+                                            <FormattedMessage id='download.scratchCards' /> -
+                                            {' '}<a href="https://cdn.scratch.mit.edu/scratchr2/static/__709da8e5f3d72129538a4ccdbcbf5f2a__/pdfs/help/Scratch2Cards.pdf">
+                                                <FormattedMessage id='download.download' />
+                                            </a>
+                                        </li>
+                                    </ul>
+                                </div>
+                            </FlexRow>
+                        </section>
+
+                        <section id="updates">
```

Comment:
I think it would be good to add a class that all the section have, and give them a small margin-bottom to separate them a bit